src/config/config.php

```diff
@@ -46,6 +46,9 @@
     // Any middleware for the graphql route group
     'middleware' => [],
 
+    // Additional route group attributes
+    'route_group_attributes' => ['guard' => 'api'],

```

Comment:
This should not be added as a default option. Add it to the comment as an example instead

Reply:
done